include/autoppl/expression/distribution/normal.hpp

```diff
@@ -41,6 +42,29 @@ struct Normal : util::DistExpr<Normal<mean_type, stddev_type>>
         return -0.5 * ((z_score * z_score) + std::log(stddev(index) * stddev(index) * 2 * M_PI));
     }
     
+    dist_value_t log_pdf_no_constant(value_t x) const
```

Comment:
I'm not sure how this is used here, but we require the pdf and log_pdf functions to support an index argument that is passed to stddev() and mean(). Does this NUTS implementation support multiple samples for its parameters?

Reply:
I removed it. It was legacy from before you split data and param so I needed this temporarily. In fact this was the source of the bug from before when I tried with multiple samples xD